src/storage.ts

```diff
@@ -247,6 +289,35 @@ export class Storage extends Service {
    */
   getBucketsStream: () => Readable;
 
+  /**
+   * Get {@link HmacKey} objects for all of the HMAC keys in the project in
+   * a readable object stream.
+   *
+   * @method Storage#getHmacKeysStream
+   * @param {GetHmacKeysOptions} [options] Configuration options.
+   * @returns {ReadableStream} A readable stream that emits {@link HmacKey} instances.
+   *
+   * @example
+   * storage.getHmacKeysStream()
+   *   .on('error', console.error)
+   *   .on('data', function(hmacKey) {
+   *     // hmacKey is an HmacKey object.
+   *   })
+   *   .on('end', function() {
+   *     // All HmacKey retrieved.
+   *   });
+   *
+   * //-
+   * // If you anticipate many results, you can end a stream early to prevent
+   * // unnecessary processing and API requests.
+   * //-
+   * storage.getHmacKeysStream()
+   *   .on('data', function(bucket) {
+   *     this.end();
+   *   });
+   */
+  getHmacKeysStream: () => Readable;
```

Comment:
Could you add this to https://github.com/googleapis/nodejs-storage/blob/0b8cbb11c9dcddff3b315eff5357655cac988909/test/index.ts#L114

Reply:
fixed!